assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -97,74 +93,70 @@ che.organization.email.org_removed_template=st-html-templates/organization_delet
 che.organization.email.org_renamed_subject=Che Organization renamed
 che.organization.email.org_renamed_template=st-html-templates/organization_renamed
 
-#####     MULTI-USER-SPECIFIC OPENSHIFT INFRASTRUCTURE CONFIGURATION       #####
-
-#     Alias of the Openshift identity provider registered in Keycloak,
-#     that should be used to create workspace OpenShift resources in
-#     Openshift namespaces owned by the current Che user.
-#     
-#     Should be set to NULL if `che.infra.openshift.project` 
-#     is set to a non-empty value.
-#
-#     For more information see the following documentation:
-#     https://www.keycloak.org/docs/3.3/server_admin/topics/identity-broker/social/openshift.html
+### Multi-user-specific openshift infrastructure configuration
 
+# Alias of the Openshift identity provider registered in Keycloak,
+# that should be used to create workspace OpenShift resources in
+# Openshift namespaces owned by the current Che user.
+# Should be set to NULL if `che.infra.openshift.project`
+# is set to a non-empty value.
+# For more information see the following documentation:
+# https://www.keycloak.org/docs/3.3/server_admin/topics/identity-broker/social/openshift.html
 che.infra.openshift.oauth_identity_provider=NULL
 
-#####                             KEYCLOACK CONFIGURATION                  #####
+#### Keycloak configuration
 
-#     Url to keycloak identity provider server
-#     Can be set to NULL only if `che.keycloak.oidcProvider`
-#     is used 
+# Url to keycloak identity provider server
+# Can be set to NULL only if `che.keycloak.oidcProvider`
+# is used
 che.keycloak.auth_server_url=http://${CHE_HOST}:5050/auth
 
-#     Keycloak realm is used to authenticate users
-#     Can be set to NULL only if `che.keycloak.oidcProvider`
-#     is used 
+# Keycloak realm is used to authenticate users
+# Can be set to NULL only if `che.keycloak.oidcProvider`
+# is used
 che.keycloak.realm=che
 
-#     Keycloak client id in che.keycloak.realm that is used by dashboard, ide and cli to authenticate users
+# Keycloak client id in che.keycloak.realm that is used by dashboard, ide and cli to authenticate users
 che.keycloak.client_id=che-public
 
-# Redhat che specific configuration
+### Redhat che specific configuration
 
-#     URL to access OSO oauth tokens
+# URL to access OSO oauth tokens
 che.keycloak.oso.endpoint=NULL
 
-#     URL to access Github oauth tokens
+# URL to access Github oauth tokens
 che.keycloak.github.endpoint=NULL
```

Comment:
Is it really RedHat Che specific configuration?
What RedHat mean here? HostedChe or something else?

Reply:
I think they are marked as `Redhat che specific configuration` because they are used in hosted che specific configuration and components.
https://github.com/eclipse/che/blob/master/multiuser/keycloak/che-multiuser-keycloak-token-provider/src/main/java/org/eclipse/che/multiuser/keycloak/token/provider/contoller/TokenController.java

Soon we are planning to make a revision of our OAuth flow and will probably remove or update this section as well. At this moment I don't think it should be a blocker to merge this pr. @sleshchenko do you agree?